nusantara/nusa_datasets/smsa/smsa.py

```diff
@@ -0,0 +1,141 @@
+from pathlib import Path
+from typing import List
+
+import datasets
+import pandas as pd
+
+from nusantara.utils import schemas
+from nusantara.utils.configs import NusantaraConfig
+from nusantara.utils.constants import Tasks
+
+_DATASETNAME = "smsa"
+_SOURCE_VIEW_NAME = "source"
+_UNIFIED_VIEW_NAME = "nusantara"
```

Comment:
can we set this to a global config variable, in a case we need to change/finalize the naming?

Reply:
@afaji : Good idea! I think dataset name is unique per dataset, so we don't need to make it as a constant, but for SOURCE_VIEW and UNIFIED_VIEW, definitely we can modify it into a constant. 

Let's just merge this PR first and I will modify all the examples altogether from the main branch.